collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -150,6 +154,7 @@ public void onCreate() {
         singleton = this;
 
         setupDagger();
+        fixGoogleBug154855417();
```

Comment:
Probably nice to add a comment with a link to the issue given this is a weird one!

Reply:
Done.